src/main/java/net/clgd/ccemux/rendering/tror/InputProvider.java

```diff
@@ -40,6 +40,7 @@ public InputProvider(InputStream stream) {
 
 				getQueue(computer).add(new InputPacket(code, payload));
 			}
+			scanner.close();
```

Comment:
Would it be worth doing `try` with resources here? Though it would be a little more verbose.